packages/vaadin-combo-box/src/vaadin-combo-box-scroller.js

```diff
@@ -273,9 +275,15 @@ export class ComboBoxScroller extends PolymerElement {
       focused: this.__isItemFocused(focusedIndex, index)
     });
 
+    el.id = `${this.__hostTagName}-item-${index}`;
     el.setAttribute('role', this.__getAriaRole(index));
     el.setAttribute('aria-selected', this.__getAriaSelected(focusedIndex, index));
-    el.setAttribute('theme', this.theme);
+
+    if (this.theme) {
+      el.setAttribute('theme', this.theme);
+    } else {
+      el.removeAttribute('theme');
+    }
```

Comment:
Since `aria-setsize` was added and virtual scrolling is used, `el.setAttribute('aria-posinset', index);` also needs to be applied so that the announced index is correct also on the items at the end of the set.

Reply:
Thanks, done.

Reply:
For some reason, the first item is announced as "1 of 200" (1-based index) but the last item announces as "199 of 200" (0-based index)

Reply:
Submitted #2747 to investigate if we can fix this later.